SCSH_3C_iOS/Persistence/CoreData/Distributor/DistributorCoreDataFacade.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  DistributorCoreDataFacade.swift
+//  Persistence
+//
+//  Created by 辜敬閎 on 2023/12/4.
+//
+
+import Combine
+
+public protocol DistributorCoreDataFacadeType {
+    func getById(id: Int64) -> AnyPublisher<Distributor, Error>
+    
+    func get() -> AnyPublisher<[Distributor], Error>
+    
+    func save(distributors: [Distributor]) -> AnyPublisher<Void, Error>
+    
+    func save(likes: [Distributor_Like]) -> AnyPublisher<Void, Error>
```

Comment:
I noticed the name `Distributor_Like` is from CoreData, right?

But don't we need another swift class or somthing else to map data names which follows the swift naming convention?

Reply:
> I noticed the name Distributor_Like is from CoreData, right?

Yes.

Each module in our projects has its own set of entities, For example, consider the 'Board' entity, as illustrated in the [instance](https://github.com/No-57/SCSH_3C_iOS/blob/main/SCSH_3C_iOS/SCSH_3C_iOS/Repository/Mapper/ExploreBoardMapper.swift), the entities need to be transformed for use within their respective module.

- SCHS_3C_iOS
- Networking
- Persistence

In the `Persistence` module, entities are generated by CoreData, with each mapped to the table name. The naming convention in this module seems appropriate. as `Persistence` can operate independently and doesn't depend on any other module like `SCHS_3C_iOS` or `Networking`.
What the table looks like would all reflect on the CoreData entity, and I used to design table naming as a snake case.

Admittedly, this approach might seem unconventional due to its deviation from typical Swift naming conventions, however, from another perspective, these names accurately mirror the underlying database tables.

Reply:
OK. I think that's just need some trade off.